dashboard/app/controllers/api_controller.rb

```diff
@@ -204,7 +204,7 @@ def user_progress_for_stage
       slog(
         tag: 'activity_start',
         script_level_id: script_level.try(:id),
-        level_id: level.id,
+        level_id: level.contained_levels.empty? ? level.id : level.contained_levels.first.id,
```

Comment:
Is there a reason we are logging only the first contained level, rather than logging all of them?